src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,42 @@
-INSERT INTO tab_transports(transport_direction,process,source_organ,source_compartment,target_organ,target_compartment,source_organ_type,source_compartment_type,target_organ_type,target_compartment_type) VALUES('InfluxBrainPlasmaToInterstitial','ActiveInfluxSpecificPlasmaToInterstitial','Brain','Plasma','Brain','Interstitial','ORGAN','COMPARTMENT','ORGAN','COMPARTMENT');
+INSERT INTO tab_categories(category,display_name,description,category_type,icon_name) VALUES('DiseaseStates','Disease States','Disease state parameters','Individual',NULL);
+INSERT INTO tab_container_names(container_type,container_name,display_name,description,extension,sequence,icon_name) VALUES('DISEASE_STATE','CKD','CKD Parameter container','CKD Parameter container',NULL,1,NULL);
+INSERT INTO tab_container_parameter_values(parameter_value_version,species,container_id,container_type,container_name,parameter_name,default_value,min_value,min_isallowed,max_value,max_isallowed,value_origin) VALUES('DiseaseStates','Human',5954,'DISEASE_STATE','CKD','InputGFR',60.0,0.0,1,60.0,1,92);

```

Comment:
this is actually wrong here. It should be 60 l/min/1.73m2. Right now it would be 60 l/min/dm2. To discuss with the definition of the dimension

Reply:
So let's define the value as `60*1.73*100`
(for the diemsnion definition - s. my comments in that PR)